src/TrackProgress/SFA.DAS.TrackProgress/SubscriptionHeaderConstants.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace SFA.DAS.TrackProgress
+{
+    public static class SubscriptionHeaderConstants
+    {
+        public const string HeaderForProviderId = "x-request-context-subscription-name";
+        public const string HeaderForSandboxMode = "x-request-context-subscription-is-sandbox";
```

Comment:
Would these strings read better if they were just ProviderId and SandboxMode, without the 'HeaderFor' prefix? - the class name identifies that they are headers.. so
[FromHeader(Name = SubscriptionHeaderConstants.ProviderId)]
instead of
[FromHeader(Name = SubscriptionHeaderConstants.HeaderForProviderId)]

Reply:
yes, I agree